kienstra/localci
# Allow injecting environment variables to jobs

#### Changes
* Allow injecting environment variables to jobs

#### Testing Steps

1. Add this to VS Code's `settings.json`:

```json
    "local-ci.command.job.enable-post-command": true
```

2. Run a job
3. You'll be prompted to enter something: <img width="685" alt="Screen Shot 2022-12-05 at 4 17 27 PM" src="https://user-images.githubusercontent.com/4063887/205755150-03dafdbb-91d7-46f7-b7cc-cca1ce317dc8.png">
4. Enter `-e FOO=BAR`
…or any other environment variable. The `-e` flag tells the CLI to expect environment variables.
5. Expected: That env var will appear in the job, redacted: <img width="699" alt="Screen Shot 2022-12-05 at 4 09 04 PM" src="https://user-images.githubusercontent.com/4063887/205755435-e5a9bb49-f534-488a-a25d-cb5e6a694840.png">
